x-pack/test/functional_synthetics/config.js

```diff
@@ -13,6 +13,13 @@ import { dockerImage as fleetDockerImage } from '../fleet_api_integration/config
 import { services } from './services';
 import { pageObjects } from './page_objects';
 
+// Docker image to use for Fleet API integration tests.
+// This hash comes from the latest successful build of the Snapshot Distribution of the Package Registry, for
+// example: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/74/pipeline/257#step-302-log-1.
+// It should be updated any time there is a new Docker image published for the Snapshot Distribution of the Package Registry that updates Synthetics.
+export const dockerImage =
```

Comment:
This has now been moved to requiring from the fleet test config so this change won't be needed